packages/lib/src/vis/raw/RawVis.tsx

```diff
@@ -8,7 +8,11 @@ interface Props {
 
 function RawVis(props: Props) {
   const { value } = props;
-  const valueAsStr = JSON.stringify(value, null, 2);
+
+  const valueAsStr =
+    value instanceof Uint8Array
```

Comment:
This could probably be made more generic for other types of `TypedArray`.

Reply:
Yeah but I didn't think the raw vis would ever receive any other kinds of typed arrays... When it's opaque, it's `Uint8Array`; h5wasm returns `Uint8Array` in a few of other cases, but otherwise it's typically a plain JS array/object. Other cases are handled by other visualizations. I didn't think it was worth making this more generic before encountering a real use case.

Reply:
Sensible :+1: